apps/frontend/src/libs/components/date-input/date-input.tsx

```diff
@@ -26,13 +26,15 @@ type Properties<T extends FieldValues> = {
 	maxDate?: Date;
 	maxRange?: number;
 	name: FieldPath<T>;
+	onPopoverClose?: (selectedDateRange: [Date, Date]) => void;
```

Comment:
i am not sure that we can do it. we should use only the `use-form` API to handle form values

Reply:
Oh, the reason I added that was because the form only submits when the button _(image)_ is clicked again. This isn't very clear from a user experience perspective, as the values change but the data isn't updated until the button is clicked. So I wanted to capture the state when the popover is closed by clicking outside the area.

Should I remove it?
![image](https://github.com/user-attachments/assets/06b37274-912f-41ab-8d92-17380bc02497)

Reply:
@liza-veis take a look please

Reply:
I think instead we need to wrap the filters block on the analytics page with a form, and whenever its value changes, submit this form. We can track it by isDirty property, or something else